templates/gitlab.yml.j2

```diff
@@ -0,0 +1,648 @@
+# # # # # # # # # # # # # # # # # #
```

Comment:
`ansible_managed`
Remove these comment blocks, just comment any non-obvious option.
I will rereview after that.

Reply:
bump

Reply:
done